Formula/xorg-server.rb

```diff
@@ -0,0 +1,70 @@
+class XorgServer < Formula
+  desc "X Window System display server"
+  homepage "https://www.x.org"
+  url "https://gitlab.freedesktop.org/xorg/xserver.git",
+    tag:      "xorg-server-21.1.3",
+    revision: "85397cc2efe8fa73461cd21afe700829b2eca768"
+  license "MIT"
+
+  depends_on "font-util"   => :build
+  depends_on "libxkbfile"  => :build
+  depends_on "meson"       => :build
+  depends_on "ninja"       => :build
+  depends_on "pkgconf"     => :build
+  depends_on "util-macros" => :build
+  depends_on "xkbcomp"     => :build
+  depends_on "xorgproto"   => :build
+  depends_on "xtrans"      => :build
+
+  depends_on "libapplewm"
+  depends_on "libxfixes"
+  depends_on "libxfont2"
+  depends_on "mesa"
+  depends_on "pixman"
+  depends_on "xcb-util"
+  depends_on "xcb-util-image"
+  depends_on "xcb-util-keysyms"
+  depends_on "xcb-util-renderutil"
+  depends_on "xcb-util-wm"
+
+  on_linux do
+    depends_on "libepoxy"
+    depends_on "libxshmfence"
+  end
+
+  def install
+    meson_args = std_meson_args + %W[
+      -Dxephyr=true
+      -Dxf86bigfont=true
+      -Dxcsecurity=true
+
+      -Dxkb_dir=#{HOMEBREW_PREFIX}/share/X11/xkb
+      -Dxkb_bin_dir=#{Formula["xkbcomp"].opt_bin}
+      -Dxkb_output_dir=#{HOMEBREW_PREFIX}/X11/xkb/compiled
+
+      -Dbundle-id-prefix=homebrew.mxcl
```

Comment:
`homebrew.mxcl` or `sh.brew`?

Reply:
What is it for?

Reply:
This option sets the bundle id of `X11.app` to ensure it doesn't conflict with other xserver app, e.g. official `XQuartz` or other user created x11 app, see [FAQ from `XQuartz`](https://www.xquartz.org/FAQs.html).
`XQuartz` program needs this id to locate the path of `X11.app`, this is also why we need `post_install` in this formula.
`xinit` also needs this bundle id to create service.